### ISSUE TITLE
What is Mailway-Android

### ISSUE THREAD

**Issue by optimumpr** [DimensionDev/Mailway-Android#1]
Sorry for opening an issue, but this is just a question: What is Mailway? Thanks in advance.

**Comment by yisiliu** [comment]
Hi @optimumpr! Thanks for asking. This is actually the successor of https://github.com/DimensionDev/Tessercube-Android, with a completely new design and better performance. In case you don't know Tessercube either, it is an input method/keyboard on mobile phones that supports end-to-end encryption while typing in your favorite instant messaging apps.

**Comment by optimumpr** [comment]
Interesting. Does this work on SMS apps?

My general recommendation would be to add special characters on long press in the main screen.

**Comment by yisiliu** [comment]
> Interesting. Does this work on SMS apps?
> 

Yes! Since it’s just an input method/keyboard, you can literally use it in any app with a text box.

> 
> My general recommendation would be to add special characters on long press in the main screen.
>

Thanks we will consider this! The eta of this app is in a month so stay tuned!
